packages/cli/src/databases/migrations/sqlite/1682411848784-SeparateExecutionData.ts

```diff
@@ -0,0 +1,74 @@
+import { logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+export class SeparateExecutionData1682411848784 implements ReversibleMigration {
+	name = 'SeparateExecutionData1682411848784';
+
+	async up({ queryRunner, tablePrefix }: MigrationContext): Promise<void> {
+		logMigrationStart(this.name);
+
+		await queryRunner.query('PRAGMA foreign_keys=OFF');
+
+		await queryRunner.query(
+			`CREATE TABLE "${tablePrefix}execution_data" (
+				"executionId" int PRIMARY KEY NOT NULL,
+				"workflowData" text NOT NULL,
+				"data" text NOT NULL,
+				FOREIGN KEY("executionId") REFERENCES "${tablePrefix}execution_entity" ("id") ON DELETE CASCADE
+			)`,
+		);
+
+		await queryRunner.query(
+			`INSERT INTO "${tablePrefix}execution_data" (
+				"executionId",
+				"workflowData",
+				"data")
+				SELECT "id", "workflowData", "data" FROM "${tablePrefix}execution_entity"
+			`,
+		);
+
+		await queryRunner.query(
+			`ALTER TABLE \`${tablePrefix}execution_entity\` DROP COLUMN "workflowData"`,
+		);
+		await queryRunner.query(`ALTER TABLE \`${tablePrefix}execution_entity\` DROP COLUMN "data"`);
+
+		await queryRunner.query('PRAGMA foreign_keys=ON');
+
+		logMigrationEnd(this.name);
+	}
+
+	async down({ queryRunner, tablePrefix }: MigrationContext): Promise<void> {
+		await queryRunner.query(`DROP TABLE IF EXISTS "${tablePrefix}temporary_execution_entity"`);
+		await queryRunner.query(
```

Comment:
could you not just alter table and add workflowData and data back in, since we do not have any of the data anyway?

Reply:
Good idea, I've changed it so we don't recreate the table.